src/routes/balances/balances.controller.spec.ts

```diff
@@ -584,47 +584,18 @@ describe('Balances Controller (Unit)', () => {
 
         expect(networkService.get.mock.calls.length).toBe(2);
       });
-
-      it(`500 error if validation fails`, async () => {
-        const chainId = '1';
-        const safeAddress = faker.finance.ethereumAddress();
-        const chainResponse = chainBuilder().with('chainId', chainId).build();
-        networkService.get.mockImplementation((url) => {
-          if (url == `${safeConfigUrl}/api/v1/chains/${chainId}`) {
-            return Promise.resolve({ data: chainResponse, status: 200 });
-          } else if (
-            url ==
-            `${chainResponse.transactionService}/api/v1/safes/${safeAddress}/balances/`
-          ) {
-            return Promise.resolve({
-              data: [{ invalid: 'data' }],
-              status: 200,
-            });
-          } else {
-            return Promise.reject(new Error(`Could not match ${url}`));
-          }
-        });
-
-        await request(app.getHttpServer())
-          .get(`/v1/chains/${chainId}/safes/${safeAddress}/balances/usd`)
-          .expect(500)
-          .expect({
-            message: 'Validation failed',
-            code: 42,
-            arguments: [],
-          });
-
-        expect(networkService.get.mock.calls.length).toBe(2);
-      });
     });
   });
 
   describe('GET /balances/supported-fiat-codes', () => {
     it('should return the ordered list of supported fiat codes', async () => {
+      const chain = chainBuilder().build();
       // So BalancesApiManager available currencies should include ['btc', 'eth', 'eur', 'usd']
-      const pricesProviderFiatCodes = ['eur', 'usd'];
+      const pricesProviderFiatCodes = ['usd', 'eth', 'eur'];
```

Comment:
We should add a test covering these in uppercase.

Reply:
Good point, added an additional test in [57c848c](https://github.com/safe-global/safe-client-gateway/pull/1175/commits/57c848c0e2e382485630f9132bda0742e2fe4b29)